settings.gradle

```diff
@@ -7,6 +7,7 @@ include 'checker-qual'
 include 'checker-qual-android'
 include 'checker-util'
 include 'framework-test'
+include 'docs'
```

Comment:
There is no `build.gradle` in the `docs` subdirectory. What does this do?

Reply:
Cleaned up this one

Reply:
I forget to commit that file. Now it is there.